aiida/orm/implementation/general/computer.py

```diff
@@ -712,9 +712,9 @@ def set_shebang(self, val):
         :param str val: A valid shebang line
         """
         if not isinstance(val, basestring):
-            raise ValueError("Input has to be a string")
+            raise ValueError("{} is invalid. Input has to be a string".format(val))
         if not val.startswith('!#'):
-            raise ValueError("A shebang line has to start with #!")
+            raise ValueError("{} is invalid. A shebang line has to start with #!".format(val))
```

Comment:
Not sure if this was intentional irony, but you actually swapped the `#` and `!` around in the conditional :smile:

Reply:
I think that if this is fixed, we still need @sphuber to accept the changes before merging?